### PR TITLE
Remove permissions for adding new instagram media.

### LINKED ISSUE #7595: Remove permissions to create Instagram posts
# Problem

- Because of the Meta API changes, we no longer have instagram add/render capabilities. When users try to add a new item, they get a cryptic error message.

# Background

https://github.com/orgs/uiowa/projects/20/views/1?pane=issue&itemId=49617004

# Proposed solution
- [x] Update permissions to prevent the creation of new Instagram posts.
- [x] Make sure this gets included in the May newsletter.
- [x] Update known issue on SiteNow site - https://sitenow.uiowa.edu/known-issues#instagram-embeds
- [x] There are also some docs linked from the known issue that will need updated.
   - [x] https://sitenow.uiowa.edu/documentation/media
   - [x] https://sitenow.uiowa.edu/documentation/media/social-media-embed
   - [x] https://sitenow.uiowa.edu/frequently-asked-questions (social media timeline question)

# Follow-up

- Consider the future of existing instagram entities and support for SiteNow. https://github.com/uiowa/uiowa/issues/7176

### PR DESCRIPTION
**Pull request by e-marie-w**
Related to #7595 

# How to test
- sync a site
- masquerade as a webmaster or lower
- try to add a new Instagram media item, via the WYSIWYG or via /media/add/. See that Instagram is no longer an option.
- Review https://sitenow.uiowa.edu/known-issues#instagram-embeds. (I just added one sentence to the end of the issue description.)

### CODE REVIEW

**Comment by GaryRidgway** [comment]
pulling to test

**Comment by GaryRidgway** [comment]
I am seeing that there is no way to add a new instagram link, but that it is still in the tabs to the left. If the team is okay with that, then I will approve after they are asked at end of day.

File: config/default/filter.format.full_html.yml
Comment (joewhitsitt) [inline config/default/filter.format.full_html.yml:1]: Not sure we need to go this far. Only admins can use full_html.

Comment (e-marie-w) [reply config/default/filter.format.full_html.yml:1]: Okay, I can remove it!

**Comment by joewhitsitt** [comment]
pulling to test

**Review by joewhitsitt — APPROVED** [review]
looks good

**Comment by e-marie-w** [comment]
Thanks @joewhitsitt!

**Pull request merged**